cmd/templates.go

```diff
@@ -66,10 +66,10 @@ mirror:
         - name: cluster-logging
           channels:
             - name: 'stable'
-  {{- if eq .Channel "4.12" }}
+  {{- if VersionAtLeast .Version "4.10" }}
```

Comment:
are we sure we want to support 4.10. If that's the case then PAO operator needs to be included. IIRC it was merged with NTO in 4.11

Reply:
Added